src/content/reference/react-dom/hydrate.md

```diff
@@ -24,59 +24,59 @@ hydrate(reactNode, domNode, callback?)
+React akan melekat pada HTML yang ada di dalam `domNode`, dan mengambil alih pengelolaan DOM di dalamnya. Sebuah aplikasi yang sepenuhnya dibangun dengan React biasanya hanya akan memiliki satu `hydrate` panggilan dengan komponen akarnya.
 
-[See more examples below.](#usage)
+[Lihat lebih banyak contoh di bawah ini.](#usage)
 
 #### Parameters {/*parameters*/}
 
-* `reactNode`: The "React node" used to render the existing HTML. This will usually be a piece of JSX like `<App />` which was rendered with a `ReactDOM Server` method such as `renderToString(<App />)` in React 17.
+* `reactNode`: Node "React" digunakan untuk me-*render* HTML yang sudah ada. Ini biasanya berupa potongan JSX seperti `<App />` yang di-*render* dengan `ReactDOM Server` metode seperti `renderToString(<App />)` di React 17.
 
-* `domNode`: A [DOM element](https://developer.mozilla.org/en-US/docs/Web/API/Element) that was rendered as the root element on the server.
+* `domNode`: Sebuah [DOM element](https://developer.mozilla.org/en-US/docs/Web/API/Element) yang di-*render* sebagai elemen root di server.
 
-* **optional**: `callback`: A function. If passed, React will call it after your component is hydrated.
+* **optional**: `callback`: Sebuah fungsi. Jika diberikan, React akan memanggilnya setelah komponen anda *hydrated*.
 
 #### Returns {/*returns*/}
 
-`hydrate` returns null.
+`hydrate` returns kosong.
 
-#### Caveats {/*caveats*/}
-* `hydrate` expects the rendered content to be identical with the server-rendered content. React can patch up differences in text content, but you should treat mismatches as bugs and fix them.
-* In development mode, React warns about mismatches during hydration. There are no guarantees that attribute differences will be patched up in case of mismatches. This is important for performance reasons because in most apps, mismatches are rare, and so validating all markup would be prohibitively expensive.
-* You'll likely have only one `hydrate` call in your app. If you use a framework, it might do this call for you.
-* If your app is client-rendered with no HTML rendered already, using `hydrate()` is not supported. Use [render()](/reference/react-dom/render) (for React 17 and below) or [createRoot()](/reference/react-dom/client/createRoot) (for React 18+) instead.
+#### Catatan {/*caveats*/}
+* `hydrate` mengharapkan konten yang di-*render* identik dengan konten yang di-*render* di server. React dapat memperbaiki perbedaan dalam konten teks, tetapi anda harus memperlakukan ketidakcocokan sebagai bug dan memperbaikinya.
+* Dalam mode pengembangan, React memberi peringatan tentang ketidakcocokan selama hydration. Tidak ada jaminan bahwa perbedaan atribut akan diperbaiki dalam kasus ketidakcocokan. Ini penting untuk alasan kinerja karena dalam sebagian besar aplikasi, ketidakcocokan jarang terjadi, sehingga memvalidasi semua markup akan menjadi sangat mahal.
```

Comment:
```suggestion
* Dalam mode pengembangan, React memberi peringatan tentang ketidakcocokan selama hidrasi (*hydration*). Tidak ada jaminan bahwa perbedaan atribut akan diperbaiki dalam kasus ketidakcocokan. Ini penting untuk alasan kinerja karena dalam sebagian besar aplikasi, ketidakcocokan jarang terjadi, sehingga memvalidasi semua markup akan menjadi sangat mahal.
```

Reply:
Shall we translate "hydration" to "*hidrasi*" and add it to the glossary, @resir014 @r17x @mhaidarhanif @mazipan?

[Hidrasi](https://kbbi.kemdikbud.go.id/entri/hidrasi) is an Indonesian word with exactly the same meaning of [hydration](https://dictionary.cambridge.org/dictionary/english/hydration) in English.